server/routes/user.details.router.js

```diff
@@ -37,18 +46,27 @@ router.get('/', rejectUnauthenticated, (req, res) => {
 /**
  * GET route for /api/user/details/id
  */
-router.get('/:id', rejectUnauthenticated, (req, res) => {
+router.get('/:id', (req, res) => {
```

Comment:
we need the security (rejectUnathenticated) back in place

Reply:
must've control z'ed that by accident XD